rlba/environments/contextual_logistic_bandit.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2022 Morteza Ibrahimi. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A Contextual Logistic Bandit environment.
+"""
+
+
+import numpy as np
+from rlba.types import (
+    Array,
+    ArraySpec,
+    BoundedArraySpec,
+    DiscreteArraySpec,
+    NestedArraySpec,
+    NestedDiscreteArraySpec,
+)
+
+
+class ContextualLogisticBandit:
+    """This class creates a contextual logistic bandit environment."""
+
+    def __init__(self, num_actions, num_contexts, dim, seed, sigma_p=1):
+        self.num_actions = num_actions
```

Comment:
I am hoping to standardize naming a bit. Could you please use `n_XYZ` instead of `num_XYZs` in new codes? For example, `n_action` instead of `num_actions`.  And please use `self._` to indicate private attributes/methods.